config/webhook/kustomizeconfig.yaml

```diff
@@ -0,0 +1,25 @@
+# the following config is for teaching kustomize where to look at when substituting vars.
```

Comment:
does this mean that we are providing a webhook now? is this required always?

Reply:
No, the webhook is _not_ enabled. This directory is generated by `operator-sdk init`. But we have not enabled it.